md/release-notes.md

```diff
@@ -111,6 +111,16 @@ In this Bonita 7.8 release, two pages have been rewritten:
  - **BPM Services** page for the profile Technical Administrator
    It has been created using the UI Designer and the new modal container. Since it is dedicated to a profile critical for the good initialization and management of Bonita, the page has not been made available in the portal's resources. It cannot be added to any application nor be edited in the UI Designer. It's designed has been slightly reviewed compared to the GWT one.
    
+<a id="performance"/>
+
+### Performance
+A few improvements have been brought to the overall Bonita efficiency:
+- Deleting an archived case went down from seconds to milliseconds. Particularly enjoyable on environments with a large amount of archived data. Moreover, deleting cases in batches is now nearly as fast as deleting a single case and is not constrained by database connexion timeouts. 
```

Comment:
ℹ️ we should change this as the improvement is only partial for now (see what we document for 7.7.5)

Reply:
ok. Changed.